README.md

```diff
@@ -28,3 +28,5 @@ Add the following require statements to your application (order is important due
     require 'newrelic_rpm'
 
 The plugin uses your application's NewRelic configuration, and will begin sending custom events for your Chore jobs. You should begin seeing data show up under "Non-Web Transactions" once this is setup.
+
+Note that the using [Chore](https://github.com/Tapjoy/chore)'s `ForkedWorkerStrategy` is incompatible with the vanilla version of the [rpm](https://github.com/newrelic/rpm). There are issues with how the child processes report data back up to their parents and this causes the child workers to die, sometimes.
```

Comment:
> Note that ~the~ using

> vanilla version of ~the~ rpm

> this causes the child workers to die, sometimes

Maybe “this can sometimes cause the child workers to die”?